Temporary workaround for gio tests failure when ran without FEATURES=userpriv
until upstream bug #552912 is fixed

--- gio/tests/live-g-file.c.orig	2008-09-25 05:44:12.848556034 +0300
+++ gio/tests/live-g-file.c	2008-09-25 06:12:34.248726237 +0300
@@ -769,11 +769,14 @@
 	      if (posix_compat)
 		{
 		  /*  target directory is not accessible (no execute flag)  */
+#if 0
+/* Fails when ran as root */
 		  do_copy_move (root, item, TEST_DIR_NO_ACCESS,
 				TEST_NO_ACCESS);
 		  /*  target directory is readonly  */
 		  do_copy_move (root, item, TEST_DIR_NO_WRITE,
 				TEST_NO_ACCESS);
+#endif
 		}
 	    }
 	}
